cmake/sdk_cmake_targets_readme.md

```diff
@@ -20,6 +20,7 @@ The targets relevant to users of the SDK are:
 
 - `openenclave::oeedger8r`
 - `openenclave::oesign`
+- `openenclave::oeapkman` (only on Linux for SGX enclaves)
```

Comment:
since you added OP-TEE support, I think we can remove the "for SGX enclaves" constraint here and anywhere else